versioned_docs/version-0.x.0/kubeslice-cli/kubeslice-cli-demo.mdx

```diff
@@ -0,0 +1,348 @@
+slice-red-ks-w-1-ks-w-2   19h
+slice-red-ks-w-2-ks-w-1   19h
+```
+
+### Validating the Slice on the Worker Cluster
+
+To validate the slice creation on each worker cluster, use the following command:
+```
+/usr/local/bin/kubectl --context=kind-ks-w-1 --kubeconfig=kubeslice/kubeconfig.yaml get slice -n kubeslice-system
+```
+Expected Output
+```
+NAME        AGE
+slice-red   19h
+```
+
+To validate the slice gateway on each worker cluster, use the following command:
+```
+/usr/local/bin/kubectl --context=kind-ks-w-1 --kubeconfig=kubeslice/kubeconfig.yaml get slicegw -n kubeslice-system
+```
+
+Expected Output
+```
+NAME                      SUBNET          REMOTE SUBNET   REMOTE CLUSTER   GW STATUS
+slice-red-ks-w-1-ks-w-2   10.190.1.0/24   10.190.2.0/24   ks-w-2
+```
+
+## Deploying the iPerf Application
+
+The `kubeslice-cli` tool sets up the iPerf demo application on the `iperf` namespace. The `iperf-server` is deployed on the `ks-w-1` worker 
+cluster and the `iperf-sleep` is deployed on the `ks-w-2` worker cluster. You need to restart the iPerf deployment to onboard the
+applications on the slice.
+
+To restart the deployment on the `ks-w-1` worker, use the following command:
+```
+/usr/local/bin/kubectl rollout  restart deployment/iperf-server -n iperf --context=kind-ks-w-1 --kubeconfig=kubeslice/kubeconfig.yaml
+```
+Expected Output
+```
+deployment.apps/iperf-server restarted
+```
+
+To restart the deployment on the `ks-w-2` worker, use the following command:
+```
+/usr/local/bin/kubectl rollout  restart deployment/iperf-sleep -n iperf --context=kind-ks-w-2 --kubeconfig=kubeslice/kubeconfig.yaml
+```
+Expected Output
+```
+deployment.apps/iperf-sleep restarted
+```
+
+## Validating the iPerf Installation
+
+To validate the iperf-server installation, use the following command:
+```
+/usr/local/bin/kubectl --context=kind-ks-w-1 --kubeconfig=kubeslice/kubeconfig.yaml get pods -n iperf
+```
+Expected Output
+```
+NAME                           READY   STATUS    RESTARTS   AGE
+iperf-server-758dd55bf-dkkbw   2/2     Running   0          36m
+```
+
+## ServiceExports and ServiceImports
+
+The iPerf server needs to be exported for visibility. Use the following command to export the iPerf server:
+
+```
+/usr/local/bin/kubectl --context=kind-ks-w-1 --kubeconfig=kubeslice/kubeconfig.yaml apply  -f kubeslice/iperf-server-service-export.yaml -n iperf
+```
+Expected Output
+```
+serviceexport.networking.kubeslice.io/iperf-server created
+```
+
+To validate the service export on the `ks-w-1` worker cluster where the iperf-server is installed, use the following command:
+```
+/usr/local/bin/kubectl --context=kind-ks-w-1 --kubeconfig=kubeslice/kubeconfig.yaml get serviceexport  -n iperf
+```
+Expected Output
+```
+NAME           SLICE       INGRESS   PORT(S)    ENDPOINTS   STATUS
+iperf-server   slice-red             5201/TCP   1           READY
+```
+
+To validate the service imports on the workers clusters, use the below commands.
```

Comment:
.. use the commands below or .. use the following commands ...

Reply:
done